README.md

```diff
@@ -94,7 +95,8 @@ some_ros_package
 |   | <b>MyNode.h</b>
 | 
 └───test
-|   | <b>my-node-test.cpp</b>
+|   | <b>my_node_test.cpp</b>
```

Comment:
As we've decided to only change the naming convention to use underscores for ros-tests, this should still be `my-node-test.cpp`